config/routes.rb

```diff
@@ -2614,7 +2614,10 @@
         show_list
         tagging_edit
         launch_configuration_profile_console
-      ]
+      ] +
+        adv_search_post +
+        exp_post +
+        save_post
```

Comment:
@DavidResende0 , I have a doubt in here ? 

shouldn't this be like this?
 ```
         tagging_edit
         launch_configuration_profile_console
         adv_search_post
         exp_post
         save_post
]
```

Reply:
No those are grouped routes coming from here https://github.com/ManageIQ/manageiq-ui-classic/blob/3926ad9c68b7c3e32def6d5dda413bbcaa546322/config/routes.rb#L10-L17